src/BrazeKit-dev.js

```diff
@@ -804,6 +925,9 @@ var constructor = function () {
 
         if (currentUser && mpid) {
             onUserIdentified(currentUser);
+            if (hasConsentMappings) {
+                prepareInitialConsent(currentUser);
+            }
```

Comment:
I think it's possible to move this if statement inside the onUserIdentified? since we already call all the time if currentUser and mpid are not null

Reply:
I know it doesn't seem like much, but I'd rather have functions that do less. Adding more and more if/else checks into functions makes this harder to understand an introduces more possible bugs in the future.

IIRC, we only need to prepare the initial consent state _once_ on init. However, `onUserIdentified` will run _every_ time an identity call is made, which can technically happen more than once during runtime.

Reply:
@alexs-mparticle is correct here about `onUserIdentified`, and I was on the same wavelength in terms of having functions do less